packages/libs/client/src/pact.ts

```diff
@@ -150,24 +183,80 @@ class PactCommand
     capability: string,
     signer: string,
     ...args: T[]
-  ): PactCommand {
+  ): this {
     const signerIndex: number = this.signers.findIndex(
       (s) => s.pubKey === signer,
     );
 
     if (signerIndex === -1) {
-      // signer not found yet
+      // signer not found
       // push new signer to this.signers
       this.signers.push({
         pubKey: signer,
         caps: [{ name: capability, args }],
       });
+      this.sigs.push(undefined);
     } else {
       // add cap to existing signer
       this.signers[signerIndex].caps.push({ name: capability, args });
     }
     return this;
   }
+
+  /**
+   * Sends a transaction to the ApiHost /local to test the transaction
+   * (i.e. it is checked whether the signatures are complete)
+   * @param apiHost - the chainweb host where to send the transaction to
+   * @alpha
+   */
+  public local(apiHost: string): Promise<ICommandResult> {
+    return local(this.createCommand(), apiHost);
+  }
+
+  /**
+   * Sends a transaction to the ApiHost /send when the transaction is finalized
+   * (i.e. it is checked whether the signatures are complete)
+   * @param apiHost - the chainweb host where to send the transaction to
+   * @alpha
+   */
+  public async send(apiHost: string): Promise<ISendResponse> {
+    const sendResponse = await send({ cmds: [this.createCommand()] }, apiHost);
+    this.requestKey = sendResponse.requestKeys[0].toString();
+    return sendResponse;
+  }
+
+  public poll(apiHost: string): Promise<IPollResponse> {
+    if (this.requestKey === undefined) {
```

Comment:
Is there a specific reason not to check on `falsy`, i.e. `!this.requestKey`?

Reply:
See comment above